message-groups.yaml

```diff
@@ -1,18 +1,12 @@
 - id: cfr
   enabled: true
   type: remote-settings
-  userPreferences:
-    - browser.newtabpage.activity-stream.asrouter.userprefs.cfr.addons
-    - browser.newtabpage.activity-stream.asrouter.userprefs.cfr.features
   frequency:
     custom:
       - {period: 86400000, cap: 1}
 - id: cfr-experiments
   enabled: true
   type: remote-settings
-  userPreferences:
```

Comment:
As testing I realized these don't do what I initially wanted. Any of those prefs disables the group. We already have the `pref` filtering as part of the targeting condition so these can be removed.

Reply:
For moments that's ok, it's only 1 pref and the result works as expected.